hydroflow/src/scheduled/mod.rs

```diff
@@ -702,3 +750,65 @@ fn test_input_handle_thread() {
 
     assert_eq!((*vec).borrow().clone(), vec![1, 2, 3]);
 }
+
+#[test]
+fn test_input_channel() {
+    // This test creates two parallel Hydroflow graphs and bounces messages back
+    // and forth between them.
+
+    use futures::channel::mpsc::channel;
+
+    let (s1, r1) = channel(8000);
+    let (s2, r2) = channel(8000);
+
+    let mut s1_outer = s1.clone();
+    let pairs = [(s1, r2), (s2, r1)];
+
+    // logger/recv is a channel that each graph plops their messages into, to be
+    // able to trace what happens.
+    let (logger, mut recv) = channel(8000);
+
+    for (mut sender, receiver) in pairs {
+        let mut logger = logger.clone();
+        std::thread::spawn(move || {
+            let done = Rc::new(RefCell::new(false));
```

Comment:
nit: `std::cell::Cell` (`.get()`/`.set()`) is canonical for `copy` types, instead of `RefCell`

Reply:
Done